include/mata/nfa.hh

```diff
@@ -1324,6 +1327,15 @@ public:
     std::list<Symbol> get_symbols() const override;
     std::list<Symbol> get_complement(const std::set<Symbol>& syms) const override;
 
+    std::string reverse_translate_symbol(const Symbol symbol) const override {
+        for (const auto& symbol_mapping: symbol_map) {
+            if (symbol_mapping.second == symbol) {
+                return symbol_mapping.first;
+            }
+        }
+        throw std::runtime_error("symbol is out of range of enumeration");
```

Comment:
I'd say it is better to print the symbol in the exception like `symbol {symbol} is out of the range of enumeration`

Reply:
Good suggestion. Fixed.